### PR TITLE
Various `README.md` improvements

### PR DESCRIPTION
**Pull request by austinkelleher**
See: https://github.com/JupiterOne/starbase/tree/readme-updates-v2#readme

## Summary

- Added logo
- Improved intro
- Added demo gif
- Simplified docs in various places

**NOTE** There are many commits in this PR. I plan to squash it all when we are happy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,115 @@
...
+  - [Whois](https://github.com/jupiterone/graph-whois)
+</details>
+
+[^1]: JupiterOne Starbase and the [Lyft Cartography](https://github.com/lyft/cartography)
+    projects compliment each other as both projects push graph data to a Neo4j
+    database instance. As such, users of Starbase can leverage the AWS connector
+    from Cartography to ingest AWS assets and relationships. A more
+    comprehensive AWS integration is used by the cloud hosted JupiterOne
+    platform and we are considering making the J1 AWS integration available as
+    open source in the future. Additionally, JupiterOne already provides a free
```
Comment (aiwilliams) [inline README.md:109]: Well, that's an exciting consideration!

File: README.md [hunk 2]
```diff
@@ -14,52 +120,110 @@ ingestion projects and optionally pushing them to a storage endpoint.
...
-See the [docs/development.md](docs/development.md) in any of the existing
-integrations for any additional details about developing other integrations.
+    These are the tools that perform data collection from third party systems and services. You can find a full list of 
+    supported [graph integrations here](https://github.com/jupiterone?q=graph-&type=all&language=&sort=). If you have a feature request, 
+    a bug to report, or you'd like to contribute to one of the supported integrations, please navigate to the specific integration repository.
+
+3. **[Graph Integrations SDK](https://github.com/jupiterone/sdk)**
+
+    The Graph Integration SDK contains core utilities and the underlying graph integration
+    runtime packages.See the [SDK development documentation](https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md)
```
Comment (aiwilliams) [inline README.md:212]:
```suggestion
    The Graph Integration SDK contains core utilities and the underlying graph integration
    runtime packages. See the [SDK development documentation](https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md)
```

File: README.md [hunk 3]
```diff
@@ -14,52 +120,110 @@ ingestion projects and optionally pushing them to a storage endpoint.
...
-See the
-[SDK development documentation](https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md)
-for a deep dive into the mechanics of how integrations work.
+2. **Graph Integrations**
 
-See the [docs/development.md](docs/development.md) in any of the existing
-integrations for any additional details about developing other integrations.
+    These are the tools that perform data collection from third party systems and services. You can find a full list of 
+    supported [graph integrations here](https://github.com/jupiterone?q=graph-&type=all&language=&sort=). If you have a feature request, 
+    a bug to report, or you'd like to contribute to one of the supported integrations, please navigate to the specific integration repository.
```
Comment (aiwilliams) [inline README.md:207]: Does this mean we're going to take care to manage Issues on GitHub?

File: README.md [hunk 4]
```diff
@@ -14,52 +120,110 @@ ingestion projects and optionally pushing them to a storage endpoint.
...
+    for a deep dive into the mechanics of how integrations work.
 
 ### Changelog
 
 The history of this project's development can be viewed at
 [CHANGELOG.md](CHANGELOG.md).
+
+### Contact
+
+Join us on `#starbase` on the [JupiterOne Community Slack](slack).
```
Comment (aiwilliams) [inline README.md:222]: 💯

File: README.md [hunk 5]
```diff
@@ -1,9 +1,115 @@
...
+- [Azure](https://github.com/jupiterone/graph-azure)
+- [Bitbucket](https://github.com/jupiterone/graph-bitbucket)
+- [GitHub](https://github.com/jupiterone/graph-github)
+- [Google Cloud](https://github.com/jupiterone/graph-google-cloud)
+- [Google Workspace](https://github.com/jupiterone/graph-google)
+- [Jira](https://github.com/jupiterone/graph-jira)
+- ...
+
+<details>
+  <summary><b>❗Click here to expand a full list of supported graph integrations❗</b></summary>
```
Comment (aiwilliams) [inline README.md:46]: How is this list going to be maintained?

**Review by aiwilliams — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (austinkelleher) [reply README.md:46]: I thought that same thing. Haven't worked that out yet 😬 . Constant list is easiest for now.

File: README.md (hunk 3 above)
Comment (austinkelleher) [reply README.md:207]: Imo, yes. Seems the most natural.

File: README.md (hunk 2 above)
Comment (austinkelleher) [reply README.md:212]: Will format in another commit. Thanks.

**Review by aiwilliams — APPROVED** [review]

**Pull request merged**